### PR TITLE
Add states for side tag updates

### PR DESCRIPTION
**Pull request by stickster**
Signed-off-by: Paul W. Frields <stickster@gmail.com>

### CODE REVIEW

File: docs/user/update_states.rst [hunk 1]
```diff
@@ -15,6 +15,14 @@ update push to be marked for stable.
 
 :ref:`stable`: The package has been released to the main updates repository.
 
+:ref:`sidetag-active`: This update represents a side tag that is modifiable.
```
Comment (pypingou) [inline docs/user/update_states.rst]: modifiable (ie: in which packagers can build packages) ?

File: docs/user/update_states.rst [hunk 2]
```diff
@@ -94,6 +102,50 @@ Bodhi will close associated bugs, and will send out update notices to the approp
...
+
+Side_tag_active
+===============
+
+An update can be created as a side tag. This corresponds to the request for a Koji side tag, which is
+a build target used to collect and iterate builds temporarily. This allows builds to be iterated without
+interfering with content in tags that ship to consumers. Once the builds are complete and correct, the
+side tag can then be merged into an existing tag.
+
+This state represents content for a side tag that is still modifiable.
```
Comment (pypingou) [inline docs/user/update_states.rst]: This state represents a side-tag whose content is still modifiable?

The ``represents content`` is what bugged me a little basically :)

**Review by pypingou — APPROVED** [review]
Couple of suggestions but nothing I'll fight over for, so :+1: for me

**Comment by stickster** [comment]
I slightly amended the text to (hopefully) clarify. Let me know if that doesn't work better for you guys.

File: docs/user/update_states.rst [hunk 3]
```diff
@@ -15,6 +15,14 @@ update push to be marked for stable.
 
 :ref:`stable`: The package has been released to the main updates repository.
 
+:ref:`sidetag-active`: This update represents a side tag whose content can still be odified; i.e. in which packages can still be built.
```
Comment (pypingou) [inline docs/user/update_states.rst]: missing a m to modified

**Review by pypingou — APPROVED** [review]
One tiny typo, otherwise :+1: for me

**Review by bowlofeggs — APPROVED** [review]
Awesome, thanks!

Even though we don't yet have side tags, I think it's OK to go ahead and merge this now. Technically Bodhi has the states even though they aren't used so I feel OK about documenting them now.

Thanks for the patch!

**Comment by bowlofeggs** [comment]
This patch is planned to be included in the upcoming [3.8.0 release](https://bodhi.stg.fedoraproject.org/docs/user/release_notes.html) and is currently deployed to [Fedora's staging Bodhi instance](https://bodhi.stg.fedoraproject.org).

**Pull request merged**